src/components/fields/image-upload/image-review/image-thumbnails/image-thumbnails.tsx

```diff
@@ -78,7 +80,7 @@ export const ImageThumbnails = (props: IProps) => {
 						<BorderOverlay isSelected={activeFileIndex === index} />
 					</ThumbnailItem>
 				);
-			} else if (image.addedFrom === "reviewModal") {
+			} else if (image.addedFrom === "reviewModal" || image.status < EImageStatus.NONE) {
```

Comment:
`image.status < EImageStatus.NONE` is not needed since there are conditions above that exclude other statuses already

Reply:
but we want it to be show the file is error when we upload multiple image 
![Uploading Screenshot 2024-04-18 at 5.36.31 PM.png…]()

Reply:
yes my point is there are conditions above that handle other statuses already, by the time the logic flow to this final if block, only the erroneous statuses are left, hence there is no need to check again

```
if (image.status === EImageStatus.NONE) { // cater for none status
	// ...
} else if (image.status > EImageStatus.NONE) { // cater for valid statuses
	// ...
} else if (image.addedFrom === "reviewModal") { // left with invalid statuses
	// ...
}
```

Reply:
no. that error only will show when u added in reviewmodal, but if add at the input there will not show the error message in the review modal.

https://github.com/LifeSG/web-frontend-engine/assets/122598226/a4363a10-fca5-4093-a928-5fb0dbf4eb06

Reply:
this is what we want

https://github.com/LifeSG/web-frontend-engine/assets/122598226/1fa6fa16-5107-481d-a627-888c4382ca10

Reply:
hello ok I get what you mean now then it's not an issue.

however I find it odd to see the error also appearing in the main image upload component just before the review confirmation modal appear. I think that error does not need to be shown, may I check if this has gone through ux?